src/backend/hip/pup/Makefile.populate_pupfns.mk

```diff
@@ -0,0 +1,43 @@
+##
+## Copyright (C) by Argonne National Laboratory
+##     See COPYRIGHT in top-level directory
+##
+## DO NOT EDIT: AUTOMATICALLY GENERATED FILE !!
```

Comment:
Should not check in this file

Reply:
I have added these `.mk` files in .gitignore, not clear how are they still getting added.

Reply:
Also `src/backend/hip/pup/Makefile.pup.mk`